src/cpu/aarch64/xbyak_aarch64/xbyak_aarch64_gen.h

```diff
@@ -5447,11 +5447,14 @@ class CodeGenerator : public CodeGenUtil, public CodeArray {
     labelMgr_.set(this);
   }
   bool hasUndefinedLabel() const { return labelMgr_.hasUndefClabel(); }
+  void clearCache(void *begin, void *end) {
+    __builtin___clear_cache((char *)begin, (char *)end);
```

Comment:
Just a note that this only works for gcc, there's a differently-named built-in for clang. Maybe something like:
```
  #if defined(__clang__)
  __clear_cache(code_addr, addr+code_len);
  #else
  __builtin___clear_cache(code_addr, code_addr+code_len);
  #endif
```

Reply:
Hi, @diaena 

Thank you for the comment.
It seems clang, at least clang version 6.0.0 or higher,  also support __builtin___clear_cache.
The below is the source of clang 6.0.0.
https://github.com/llvm-mirror/clang/blob/release_60/include/clang/Basic/Builtins.def#L470

Reply:
Oh sorry, nice then!

Reply:
Hi @kawakami-k , it seems this intrinsics is actually supported since [clang 3.8](https://github.com/llvm-mirror/clang/blob/07a6361e0f32f699d47c124106e7911b584974d4/include/clang/Basic/Builtins.def#L450), which is the minimal requirement for oneDNN. So this change should be good, thanks.